v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/templates/KafkaToBigQueryFlex.java

```diff
@@ -198,30 +200,25 @@ public static PipelineResult run(KafkaToBigQueryFlexOptions options)
      *  4) Write failed records out to BigQuery
      */
 
-    ImmutableMap<String, Object> kafkaConfig =
+    ImmutableMap<String, Object> kafkaConfig = ImmutableMap.<String, Object>builder().build();
+
+    kafkaConfig =
         ImmutableMap.<String, Object>builder()
-            .putAll(KafkaCommonUtils.configureKafkaOffsetCommit(options))
+            .putAll(kafkaConfig)
+            .putAll(KafkaConfig.fromReadOptions(options))
```

Comment:
I think this whole block can be replaced with `Map<String, Object> kafkaConfig = KafkaConfig.fromReadOptions(options)`?

`.putAll(kafkaConfig)` currently doesn't do anything because `kafkaConfig` is an empty map, if I'm reading this correctly.

Reply:
Done